docs/pages/reference/signature-algorithms.mdx

```diff
@@ -0,0 +1,223 @@
+---
+title: Signature Algorithms
+h1: Signature Algorithms Reference
+description: "Signature algorithms used in Teleport."
+---
+
+An important role of the Teleport Auth Service is issuing short-lived SSH and
+TLS certificates to users and hosts that allow all connections to be
+authenticated, authorized, and encrypted.
+This page describes the various cryptographic signature algorithms used to sign
+each kind of certificate issued by Teleport.
+
+Historically, every Teleport-issued certificate was signed by a 2048-bit RSA key.
+RSA is one of the oldest and most well-known asymmetric cryptography algorithms.
+While it is still considered secure (as long as your keys are large enough),
+modern alternatives based on Elliptic-Curve Cryptography offer much better
+performance with smaller keys and better security.
```

Comment:
Since the algorithms are configurable, I would use this paragraph to describe when a user would want to configure the algorithm used, i.e., when they would want to continue reading in order to choose an algorithm to configure (answering the question, "Does this guide apply to my use case?").

Reply:
edited the full intro to:
> The Teleport Auth Service issues SSH and TLS certificates to users and hosts that allow all connections to be authenticated, authorized, and encrypted. This page describes the cryptographic signature algorithms used to sign each kind of certificate issued by Teleport.
>
> Continue reading to learn how to:
>
> - configure a Teleport cluster created before Teleport 17 to use fast and secure elliptic-curve keys
> - configure your cluster to use FIPS-compatible signature algorithms
> - configure your cluster to use signature algorithms compatible with your HSM or KMS.